[Docs] modify README.md to match modified props (#152) (#159)
modern-agile-team/dev-portfolio: README.md

};
```

- channels example

<img width="140" alt="스크린샷 2022-10-01 오후 1 55 33" src="https://user-images.githubusercontent.com/56839474/193393251-a182a876-956d-4ccf-a18d-b0f4a713ad3e.png">

> Examples are separate. See the corresponding <a href="#more-about-channels-props">Link.</a>

Reviewer: ```suggestion
For an example of channels, see: <a href="#more-about-channels-props">channels example</a>
```
